watch_faces/BinarySystem/source/DataViews/BatteryView.mc

```diff
@@ -7,12 +7,33 @@ using Toybox.Math as Math;
 
 class BatteryView extends Ui.Drawable
 {
+
+    var showBatteryBar = true;
+    var showBatteryPercentage = true;
+    var batteryBarHorizontal = true;
+
+    var batHist = {};
+    var batHistCount = 0;
+    var timeInterval = 0;
+    var remainingBattery;
+
+    var batteryBarSize = 50;
+    var batteryBarThickness = 2;
+    var batteryBarLocX= 109;
+    var batteryBarLocY = 50;
+
+    var batteryPercentageLocX = 109;
+    var batteryPercentageLocY =10;
+
+    var fg_color = Gfx.COLOR_WHITE;
+    var bg_transp = Gfx.COLOR_TRANSPARENT;
+        var dot_color = App.getApp().getProperty("ForegroundColor");
+
+
     function initialize(params)
     {
         Drawable.initialize(params);
 
-        showBatteryBar = params.get(:showBatteryBar);
```

Comment:
as far as ive seen the layout settings are loaded at first and if the user changes something it will take his new values, but we probably need to test this in detail